### PR TITLE
Lesson 38

### PR DESCRIPTION
**Pull request by LudmilaZhuleha**
Added Routes

### CODE REVIEW

File: src/components/Card/Card.jsx [hunk 1]
```diff
@@ -1,8 +1,11 @@
 import React from "react";
 import PropTypes from 'prop-types';
 import './Card.css';
+import {useParams} from 'react-router-dom'
 
 function Card(data) {
+  // const { id } = useParams();
```
Comment (Marinarium) [inline src/components/Card/Card.jsx]: please delete comments

File: src/components/Card/Card.css [hunk 2]
```diff
@@ -4,10 +4,12 @@
     flex-basis: 23%;
     list-style: none;
     margin-bottom: 5px;
+    cursor: pointer;
+    text-decoration: none;
 }
 
 .card-image {
-    width: 100%;
+    width: 296px;
```
Comment (Marinarium) [inline src/components/Card/Card.css]: it's better to use max-widh if it's possible

File: src/components/Footer/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import './Footer.css'
+
+const Footer = () => {
+  const descriptions =['How Triphouse works','Careers','Privacy','Terms'];
+  const homesTypes =['Guest houses','Hotels','Apartments','Villas','Holiday homes','Hostels'];
+
+  const descriptionList = descriptions.map((item, i)=>{
+    return (<li className="footer-item" key={i}>{item}</li>)});
+
+  const homesTypesList = homesTypes.map((item, i)=>{
```
Comment (Marinarium) [inline src/components/Footer/Footer.jsx]: descriptionList and homesTypesList  look the same. can you use a component for them?

File: src/components/Card/Card.jsx (hunk 1 above)
Comment (LudmilaZhuleha) [reply src/components/Card/Card.jsx]: Done

File: src/components/Card/Card.css (hunk 2 above)
Comment (LudmilaZhuleha) [reply src/components/Card/Card.css]: Added max-width

File: src/components/Footer/Footer.jsx (hunk 3 above)
Comment (LudmilaZhuleha) [reply src/components/Footer/Footer.jsx]: Added List component.

**Pull request merged**